src/discrete_rv.jl

```diff
@@ -68,13 +68,13 @@ Make multiple draws from the discrete distribution represented by a
 
 - `out::Vector{Int}`: `k` draws from `d`
 """
-Base.rand(d::DiscreteRV, k::Int) = Int[rand(d) for i=1:k]
+Random.rand(d::DiscreteRV, k::Int) = Int[rand(d) for i=1:k]
 
-function Base.rand!(out::AbstractArray{T}, d::DiscreteRV) where T<:Integer
+function Random.rand!(out::AbstractArray{T}, d::DiscreteRV) where T<:Integer
     @inbounds for I in eachindex(out)
         out[I] = rand(d)
     end
     out
 end
 
-@deprecate draw Base.rand
+@deprecate draw Random.rand
```

Comment:
I think we can now drop this deprecation -- it has been here for a while and when we release QuantEcon.jl again it will be a new minor version, which gives us room to make breaking changes.